AEPServices/Sources/ui/UIUtils.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ Copyright 2021 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+import UIKit
+
+class UIUtils {
```

Comment:
Something i am running into while writing tests for the preview manager is that any business logic which relies on the results from `UIUtils` will fail because the unit tests cannot produce a frame. We should make this class mockable in my opinion and it should either be injected into the UI elements (fullscreen message, alert, button), or it should be part of the service provider. Any thoughts on this? @shalehaha @nporter-adbe

Reply:
Agreed, if the results of this class need to be used in unit tests somewhere then we should define UIUtils to conform to a protocol and injected for easy testing.